.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      actions:
+        patterns: ["*"]
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)